projects/canopy/src/lib/forms/date/date-field.component.scss

```diff
@@ -16,6 +16,10 @@
 
   & .lg-input-field .lg-input-field__inputs {
     width: 100%;
+
+    & .lg-input {
```

Comment:
The only thing I'd say is that I don't think we are supporting IE11 anymore in Canopy so should this fix be here?
@owensgit @andy-polhill ?

Reply:
@elenagarrone I spoke to the Product Team, got confirmation that we don't need to support IE11. I will go ahead and close PR.